.github/workflows/propagate-doc-upwards.yml

```diff
@@ -55,7 +55,7 @@ jobs:
           ./.././bonita-documentation-site/scripts/propagate_doc_upwards.bash
 
       - name: Send message to Slack channel
-        if: failure() && (github.event_name == 'schedule' || (github.event_name == 'workflow_dispatch' && github.event.inputs.slack-notifications == 'true') )
```

Comment:
**question:** did you test the change by running the action? AFAIK, whatever the type is defined in the input, the actual type at runtime is always a string, so this change may change the behaviour (the input is always set, so the condition would always be true)

Reply:
My bad, reverting it...